ansible/playbooks/nuke.yml

```
@@ -10,6 +10,10 @@
     default: "NO"
     private: no
   any_errors_fatal: yes
+  vars:
+    uninstall: true
```

I think we refactor the variable name to be unique for the role - if we do it here we'll also need to add it to other instances where this var is referenced.

```suggestion
    cri_uninstall: true
```